content/karpenter/050_karpenter/consolidation.md

```diff
@@ -245,7 +245,7 @@ Run the following command to set the number of replicas to 6
 kubectl scale deployment inflate --replicas 3
 ```
 
-As of version 0.16.1, Karpenter only uses the **Delete** consolidation mechanism for spot nodes. It will not replace a spot node with a cheaper spot node. Spot instance types are selected with the `capacity-optimized-prioritized` strategy and often the cheapest spot instance type is not launched due to the likelihood of interruption. Consolidation would then replace the spot instance with a cheaper instance negating the `capacity-optimized-prioritized` strategy entirely and increasing interruption rate.
+For spot nodes, Karpenter only uses the deletion consolidation mechanism. It will not replace a spot node with a cheaper spot node. Spot instance types are selected with the `price-capacity-optimized` strategy and often the cheapest spot instance type is not launched due to the likelihood of interruption. Consolidation would then replace the spot instance with a cheaper instance negating the `price-capacity-optimized` strategy entirely and increasing interruption rate.
```

Comment:
I'd keep the `**Delete**` word as it makes reference to one of the actions `consolidation` could take.

Reply:
Is it okay to rename to Deletion as this is what is in the Karpenter docs? https://karpenter.sh/v0.23.0/concepts/deprovisioning/#consolidation